File: 01-Basics/07-Palindrome-Number/Palindrome.java
public class Palindrome {
    public static void main(String[] args) {
        int original = 12321;
        int num = original;
        int reversed = 0;

        while (num > 0) {
            int rem = num % 10;
            reversed = reversed * 10 + rem;
            num = num / 10;
        }

        if (original == reversed) {
            System.out.println("Palindrome");
        } else {
            System.out.println("Not Palindrome");
        }
    }
}

File: 01-Basics/07-Palindrome-Number/palindrome.py
original = 12321
num = original
reversed_num = 0

while num > 0:
    rem = num % 10
    reversed_num = reversed_num * 10 + rem
    num = num // 10

if original == reversed_num:
    print("Palindrome")
else:
    print("Not Palindrome")